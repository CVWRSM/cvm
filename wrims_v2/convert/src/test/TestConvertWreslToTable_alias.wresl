define QsacFth {alias C_SacFea+D_SacFea  kind 'flow-channel' units 'cfs'}
define CvpTotalDel {alias D700+D710+D706+D607A+D833+D835+D836+D839+D841+D855+D837+D843+ ! AG
                          D701+D707+D607B+ ! EX
                          D711+D844+ ! MI
                          D708+D607C_PRF+D856_PRF+ ! RF
                          D702+D607D+D834+D838+D840+D842+D845 ! LS
                          kind 'FLOW-DELIVERY' units 'CFS' }