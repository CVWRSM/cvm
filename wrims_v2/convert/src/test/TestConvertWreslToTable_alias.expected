define QsacFth {alias C_SacFea+D_SacFea  kind 'flow-channel' units 'cfs'}
